src/main/java/com/lberkholtz/advancedava/week3/Main/StockQuoteApplication.java

```diff
@@ -43,7 +60,7 @@ public static void main(String[] args) throws ParseException {
                 startDate.setTime(startdateformat.parse(fromDate));
                 endDate.setTime(enddateformat.parse(untilDate));
 
-            } catch (ParseException e) {
+                } catch (ParseException e) {
 
                 e.printStackTrace();
```

Comment:
yikes, there is that unfortunate error handling (or actually the lack of it). 
Let me know if you need help here understanding how to handle exceptions correctly

Reply:
I ended up changing it to this so the program terminates with an error message. If there was input from the console I could ask for a different input, but we don't have that in this application, so I figured terminating with an error was best.  Let me know if this is OK.

                } catch (ParseException e) {

                System.out.println("Invalid dates. Program is terminating");
                System.exit(-1);

            }